src/examples/anchor/AnchorDanger.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Anchor } from '@zendeskgarden/react-buttons';
+
+const Example = () => {
+  return (
+    <Anchor isDanger href="#">
```

Comment:
@rossmoody I think it would be nifty if these examples linked to their respective sections. So, in this case `href="#danger"` – would help to prevent context jolt. Also, is there an opportunity to use copy other than "Go to the Garden"? (which doesn't sound very dangerous :wink:).

Reply:
Love both points. @vsri1137 I'll take a stab, but if you weigh in here, we can update.